test/bert_test.rb

```diff
@@ -57,6 +56,13 @@ class BertTest < Test::Unit::TestCase
       end
     end
 
+    should "do roundtrips for date time" do
+      now = Time.now
```

Comment:
I read somewhere recently (might be rubocop) that time using timezone (like `Time.zone.now`)was a better option, does it apply here?

Reply:
It does only in Rails projects. Ruby doesn't have `Time#zone`

Reply:
The rationale is that using Time without specifying the zone is using server tz or TZ= env which might be different from the Time.zone= set in rails projects.

Reply:
right, my bad, thanks :)